config/actions/setup-runner/action.yml

```diff
@@ -27,12 +27,25 @@ runs:
       run: |
         echo "JAVA_HOME=$(/usr/libexec/java_home --version ${JDK_VERSION})" >> $GITHUB_ENV
       shell: bash
+    - name: Download homebrew dependencies
+      run: |
+        if ! which brew; then
+          bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+        fi
+        brew bundle --no-lock
+      shell: bash
 
-    - name: Install snap dependencies
+    - name: Install Gemfile dependencies
       run: |
-        sudo snap install vale --edge
+        export LC_ALL=en_GB.UTF-8
+        export LANG=en_GB.UTF-8
+        export GEM_HOME=$HOME/.gem
+        export GEM_PATH=$HOME/.gem
+        eval "$(rbenv init - bash)"
+
+        rbenv install --skip-existing
```

Comment:
`rbenv` won't know what version of ruby to run so you need a `.ruby-version` file so it can.

This file just contains the version number of ruby you're looking for. Within the ID Check space, this is just `3.3.0`.

Reply:
Originally I tried installing `3.3.0` with apt, and struggled as the version was not available in the registry. The reason why I am trying to move to `brew` is to circumvent that issue